src/components/CandidateAvatar/__tests__/CandidateAvatar.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import CandidateAvatar from '../CandidateAvatar';
+
+describe('<CandidateAvatar> that renders a candidate profile image with match indication', () => {
+    it('should render a default avatar', () => {
+        const wrapper = shallow(<CandidateAvatar />);
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it('should render a custom sized avatar with good match percentage', () => {
+        const wrapper = shallow(<CandidateAvatar size={128} percentage={67} />);
+        expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
instead of just adding a snapshot, could you add a specific test if what to look for in the snapshot? like `wrapper.find('circle').hasClass('CandidateAvatar__circle--context_bad')`. That would make the test more future proof. 
The same for all the following tests.

Reply:
Done - but in some cases a snapshot really is more convenient: the circle `stroke-dasharray` computation is a bit complex and results in a number that you can't easily derive from `size`